### DISCUSSION TITLE
Release schedule?

### DISCUSSION THREAD

**Discussion by buckaroogeek** [ansible-community/molecule-plugins#220] [General]
Greetings

Is there a release schedule available? There are recent updates to the podman plugin I am very interested in.

thanks

**Comment by zhan9san** [comment]
plz wait for one more pr
openstack plugin.

maybe in two weeks

**Reply by buckaroogeek** [reply]
Thank you for the update. Much appreciated. And thank you for work on these plugins.

**Comment by zhan9san** [comment]
@ajinkyau 

Could you please create a release?

**Reply by zhan9san** [reply]
Hi @ajinkyau  and @ssbarnea 

Sorry to bother you.

I tried to draft a release, but it failed.

Could you help me?


Here is what I did.
1. Click "Draft` a release" in this page, https://github.com/ansible-community/molecule-plugins/releases
2. Click "Choose a tag" and type v23.5.2
3. Click "Generate release notes"(The format of release note is different from the previous one, do you know why)
4. Check "Create a discussion for this release "
5. Click "Publish release"

The tag was created successfully, but the release action failed.

Is it permission issue? Or did do anything wrong?

Here is the failed action run, https://github.com/ansible-community/molecule-plugins/actions/runs/7529883395
